scripts/install-worker.sh

```diff
@@ -240,41 +240,54 @@ sudo mkdir -p /var/lib/kubernetes
 sudo mkdir -p /var/lib/kubelet
 sudo mkdir -p /opt/cni/bin
 
-echo "Downloading binaries from: s3://$BINARY_BUCKET_NAME"
-S3_DOMAIN="amazonaws.com"
-if [ "$BINARY_BUCKET_REGION" = "cn-north-1" ] || [ "$BINARY_BUCKET_REGION" = "cn-northwest-1" ]; then
-  S3_DOMAIN="amazonaws.com.cn"
-elif [ "$BINARY_BUCKET_REGION" = "us-iso-east-1" ]; then
-  S3_DOMAIN="c2s.ic.gov"
-elif [ "$BINARY_BUCKET_REGION" = "us-isob-east-1" ]; then
-  S3_DOMAIN="sc2s.sgov.gov"
-fi
-S3_URL_BASE="https://$BINARY_BUCKET_NAME.s3.$BINARY_BUCKET_REGION.$S3_DOMAIN/$KUBERNETES_VERSION/$KUBERNETES_BUILD_DATE/bin/linux/$ARCH"
-S3_PATH="s3://$BINARY_BUCKET_NAME/$KUBERNETES_VERSION/$KUBERNETES_BUILD_DATE/bin/linux/$ARCH"
-
-BINARIES=(
-  kubelet
-  aws-iam-authenticator
-)
-for binary in ${BINARIES[*]}; do
-  if [[ -n "$AWS_ACCESS_KEY_ID" ]]; then
-    echo "AWS cli present - using it to copy binaries from s3."
-    aws s3 cp --region $BINARY_BUCKET_REGION $S3_PATH/$binary .
-    aws s3 cp --region $BINARY_BUCKET_REGION $S3_PATH/$binary.sha256 .
-  else
-    echo "AWS cli missing - using wget to fetch binaries from s3. Note: This won't work for private bucket."
-    sudo wget $S3_URL_BASE/$binary
-    sudo wget $S3_URL_BASE/$binary.sha256
+if [ "$PULL_K8S_FROM_UPSTREAM" = "true" ]; then
+  echo "Downloading binaries from upstream"
+  sudo wget "https://dl.k8s.io/release/v$KUBERNETES_VERSION/bin/linux/amd64/kubelet"
+  sudo wget "https://dl.k8s.io/release/v$KUBERNETES_VERSION/bin/linux/amd64/kubelet.sha256"
+  echo "$(cat kubelet.sha256) kubelet" | sha256sum -c -
+  sudo chmod +x kubelet
+  sudo mv kubelet /usr/bin/
+
+  wget -O aws-iam-authenticator "https://github.com/kubernetes-sigs/aws-iam-authenticator/releases/download/v0.6.2/aws-iam-authenticator_0.6.2_linux_amd64"
```

Comment:
Hm, I think it's kind of odd that we store `aws-iam-authenticator` and `ecr-credential-provider` under each Kubernetes semver + build date prefix in the S3 bucket. Those dependencies should probably be separate (and therefore configurable).